frontend/src/pages/goals.tsx

```
@@ -50,19 +70,29 @@ export default function Goals() {
-                </div>
-                <div className={styles.containerRow}>
-                    <div>
+                    <div className={styles.goalGroup}>
                         <h2>Monthly Goals</h2>
                         <GoalsList goals={monthlyGoals} />
                     </div>
-                    {/* <h2>Calendar</h2> */}
+                </div>
+                <div className={styles.goalRightColumn}>
+                    {showPersonalGoalButton && (
+                        <button onClick={() => setShowGoalSettingModal(true)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)